include/indexstore/indexstore.h

```diff
@@ -53,9 +53,13 @@
 
 #ifndef INDEXSTORE_PUBLIC
 # if defined (_MSC_VER)
-#  define INDEXSTORE_PUBLIC __declspec(dllimport)
+#  if defined(_CINDEX_LIB_)
```

Comment:
`_CINDEX_LIB_` came from mindlessly copy-pasting the CMake configuration from libclang.
This should be its own name, like `_CINDEXSTORE_LIB_`.

Reply:
Ah, okay, I'll fix that.